students/jemielity_kamil/lesson_05_lists/maximum.py

```diff
@@ -0,0 +1,19 @@
+rows_and_columns = [int(i) for i in input('Numbers: ').split()]
+m = rows_and_columns[0]  # rows
+n = rows_and_columns[1]  # columns
+
+array = [[int(j) for j in input('Numbers in row: ').split()] for k in range(m)]
+
+maximum = array[0][0]
+
+for i in range(len(array)):
+    for j in range(len(array[0])):
+        if array[i][j] > maximum:
+            maximum = array[i][j]
+
+for i in range(len(array)):
+    try:
+        print(str(i) + ' ' + str(array[i].index(maximum)))
+        break
+    except ValueError:
```

Comment:
Please avoid handling exception if it's not necessary. It's better to add a condition for checking the index.
Anyway, i and j are already known in the upper code, consider the following:
```
rows_and_columns = [int(i) for i in input('Numbers: ').split()]
m = rows_and_columns[0]  # rows
n = rows_and_columns[1]  # columns

array = [[int(j) for j in input('Numbers in row: ').split()] for k in range(m)]

maximum_i = 0
maximum_j = 0
maximum = array[maximum_i][maximum_j]


for i in range(len(array)):
    for j in range(len(array[0])):
        if array[i][j] > maximum:
            maximum = array[i][j]
            maximum_i = i
            maximum_j = j

print('{} {}'.format(maximum_i, maximum_j))

```